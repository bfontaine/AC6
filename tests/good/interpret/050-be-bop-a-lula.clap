def  be   (x) = x
with bop  (x) = x
with a    (x) = x
with lula (x) = x

val music = be.bop.a.lula

val p = music "please"
