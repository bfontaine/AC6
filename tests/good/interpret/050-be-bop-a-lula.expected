:- be = <code>
:- bop = <code>
:- a = <code>
:- lula = <code>
:- music = <code>
:- p = "please"
